static/js/analytics.js

```diff
@@ -313,6 +322,7 @@ const renderHashtagsGraphFromJson = (json) => {
 };
 
 const renderGeotaggedMapFromJson = (json) => {
+  if (!document.getElementById("chart-geotagged")) return;
```

Comment:
nit: 

Good idea, but could you make it more similar to the check we use above? 

One line for the document.getElementById,
check on next line and wrapped by curly pharenthesis

Reply:
Done